peer_mentoring/posts/models.py

```diff
@@ -16,7 +16,7 @@ class Post(models.Model):
     author = models.ForeignKey(User, on_delete=models.CASCADE)
     title = models.CharField(max_length=150)
     content = models.TextField(blank=False, null=True)
-    comments = models.OneToMany(Comment)
+    comments = models.ForeignKey(Comment, on_delete=models.CASCADE)
```

Comment:
ignore my previous commit comment, you already found it, good!

Reply:
but wait it seems you did it the other way around, I would expect the foreign key to go on the comment table back to post? 

so:

```
class Comment(models.Model):
    ... fields ...
    post = models.ForeignKey(Post, on_delete=models.CASCADE)
```

Reply:
Yup, taking care of it. Glad to get ahead of you, but my bad on the wrong class hah.